src/libs/actions/Task.js

```diff
@@ -821,6 +825,10 @@ function cancelTask(taskReportID, taskTitle, originalStateNum, originalStatusNum
     ];
 
     API.write('CancelTask', {cancelledTaskReportActionID: optimisticReportActionID, taskReportID}, {optimisticData, successData, failureData});
+
+    if (shouldDeleteTask) {
```

Comment:
The 'cancel' word in the header options is renamed to 'delete' so irrespective of whether there is a last message we call this action deleting the task I think. So this wording `shouldDeleteTask` kinda appears confusing. Maybe `shouldDeleteTaskReport` is better?